packages/celotool/src/lib/geth.ts

```diff
@@ -468,126 +467,168 @@ const measureBlockscout = async (
   }
 }
 
+export const transferCeloGold = async (
+  kit: ContractKit,
+  fromAddress: string,
+  toAddress: string,
+  amount: BigNumber,
+  txOptions: {
+    gas?: number
+    gasPrice?: string
+    gasCurrency?: string
+    gasFeeRecipient?: string
+  } = {}
+) => {
+  const kitGoldToken = await kit.contracts.getGoldToken()
+  return kitGoldToken.transfer(toAddress, amount.toString()).send({
+    from: fromAddress,
+    ...txOptions,
+  })
+}
+
+export const transferCeloDollars = async (
```

Comment:
I know you just copied it, but you could probably just have a single `transferToken` function instead of having to duplicate the methods here